api/apps/articles/routes.coffee

```diff
@@ -1,43 +1,56 @@
 _ = require 'underscore'
 { present, presentCollection } = Article = require './model'
 { setUser } = require '../users/routes'
+User = require '../users/model.coffee'
 
 # GET /api/articles
 @index = (req, res, next) ->
-  if req.query.published isnt 'true' and (not req.query.author_id? or
-     req.query.author_id isnt req.user?._id.toString()) and
-     req.user?.type isnt 'Admin'
-    return res.err 401,
-      'Must pass author_id=me to view unpublished articles. Or pass ' +
-      'published=true to only view published articles.'
+  return res.err(401, 'Must pass channel_id to view unpublished articles. Or pass ' +
```

Comment:
Nice! The early returns definitely help following the auth logic here. Small style note: it might be easier to read by putting the `if`s first, and/or pulling the messages out into variables e.g. `res.err(401, messages.requiredParams) if ...`.